client/src/components/FormResponses/FormResponses.tsx

```diff
@@ -390,12 +405,18 @@ const FormResponses = ({ formId }: FormResponsesProps) => {
               <td>{email[index]}</td>
               <td>{`$${formattedCurrency[index]}`}</td>
               <td>
-                {paymentType[index] === "Credit Card / Stripe"
-                  ? getStatusOfStripePayment(
+                {paymentType[index] === "Credit Card / Stripe" ? (
```

Comment:
I really wish `"Credit Card / Stripe"` was a payment type that we can share 🙏 

overall a good structure that I would have liked to see is ...
```suggestion
                {payment.type === "Credit Card" ? (
```

Reply:
Great suggestion I think for now we can defer this to another PR